profile/README.md

```diff
@@ -23,7 +23,9 @@ In addition to running standalone, Project CodeFlare is deployed as part of and
 ### Learning
 
 Watch [this video][youtube-demo] for an introduction to Project CodeFlare and what the
-stack can do.
+stack can do. (Nov. 2022)
+
+See [this video][summit-demo] as well for an updated demonstration of the basic stack functionality in-action. (Jun. 2023)
```

Comment:
Should we just keep this as the main demo to look at? And effectively just remove the old one?

Reply:
I like this idea.

Reply:
The only thing with the old demo is that it has a bit of explanation/presentation before it. Would keeping that be useful, or is it not worth the confusion of having two different videos?

Reply:
I just took a look at both and there is something to be said about the old demo. It gives a explanation of the "why" of the stack as @Maxusmusti said. The new one just jumps straight into the demo. I feel the first one does a better job of "selling" the stack.